src/databricks/labs/ucx/queries/assessment/main/01_4_table_crawl_failures.sql

```diff
@@ -0,0 +1,23 @@
+/*
+--title 'Table Crawl Failures'
+--height 4
+--width 4
+*/
+WITH table_crawl_failures AS (
+SELECT
+    timestamp,
+    REGEXP_EXTRACT(message, '^failed-table-crawl: (.+?) -> (.+?): (.+)$', 1) AS error_reason,
+    REGEXP_EXTRACT(message, '^failed-table-crawl: (.+?) -> (.+?): (.+)$', 2) AS error_entity,
+    REGEXP_EXTRACT(message, '^failed-table-crawl: (.+?) -> (.+?): (.+)$', 3) AS error_message,
+FROM inventory.logs
+  WHERE
```

Comment:
narrow down to the latest run of `crawl_tables` task, otherwise it'll show the outdated results. Add a screenshot to confirm.